nexus/reconfigurator/preparation/src/lib.rs

```diff
@@ -72,3 +89,147 @@ pub fn policy_from_db(
 
     Ok(Policy { sleds, service_ip_pool_ranges, target_nexus_zone_count })
 }
+
+/// Loads state for import into `reconfigurator-cli`
+///
+/// This is only to be used in omdb or tests.
+pub async fn reconfigurator_state_load(
```

Comment:
This was basically copied wholesale out of `omdb` because I needed it in the new reconfigurator-cli test.

Longer term we might want to combine this with the new `PlanningInput` but that doesn't exist yet.